_posts/2020-09-17-anna-krystalli.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "R for Reproducibility"
+author: "@pweschmidt"
+rse: "Anna Krystalli"
+phenotype: 
+excerpt: "What do plankton surveys, R and reproducibility have in common? In this episode, Anna Krystalli from the University of Sheffield (UK) shares her vast experience and covers areas from macro biology to R programming, community building to making software more open and reproducible."
+date: 2020-09-17 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-anna-krystalli-episode-34.mp3
```

Comment:
```suggestion
external_media: https://github.com/USRSE/rse-stories-episodes-1/raw/master/2020/rse-stories-anna-krystalli-episode-34.mp3
```
External media for other link doesn't seem to be working, will try raw file link.